docs/advanced/debugging.rst

```diff
@@ -3,16 +3,36 @@ Debugging
 =========
 
 Logging
-=======
+-------
 
-The Flocker processes running on the nodes will write their logs to ``/var/log/flocker/``.
-The log files are named ``<processname>-<pid>.log``, e.g. ``flocker-volume-1234.log``.
+Flocker processes generally use `eliot`_ for logging.
```

Comment:
is "generally" necessary? (or I guess it should be "mostly" but is that true or is it "always"?).